src/module/constElements.js

```diff
@@ -2,17 +2,11 @@ const refreshTask = document.querySelector('#refresh');
 const taskInput = document.querySelector('#task');
 const submitInput = document.querySelector('#arrow-submit');
 const taskList = document.querySelector('#taskList');
-const clearCompletedTasks = document.querySelector('#clear-completed');
+// const clearCompletedTasks = document.querySelector('#clear-completed');
```

Comment:
- [ ] Please, remove this old piece of commented code so the repo gets cleaner 💪